cmd/webshorten/main.go

```diff
@@ -47,9 +51,23 @@ func shortenURL(w http.ResponseWriter, r *http.Request) {
 	w.Write(js)
 }
 
+var flagPort *int = flag.Int("port", 0, "use '-port=NUMBER' flag to run server on specific port(default: 5000)")
```

Comment:
There are no need to specify a type `*int` here. Let compiler inference it. 
```suggestion
var flagPort = flag.Int("port", 0, "use '-port=NUMBER' flag to run server on specific port(default: 5000)")
```